scripts/locations_data_load/test/events/response.json

```diff
@@ -0,0 +1,225 @@
+{
+  "resourceType": "Bundle",
+  "id": "c6013fb0-a50f-4328-bb0d-1e78d3de6c05",
+  "meta": {
+    "lastUpdated": "2024-01-11T18:15:21.030+00:00"
+  },
+  "type": "searchset",
+  "total": 7,
+  "link": [ {
+    "relation": "self",
+    "url": "https://beta.ods.dc4h.link/fhir/Organization?active=true&type=RO209"
```

Comment:
We'll need to have a think about how we shift the URLs from here

Reply:
removed this file completely